src/Update.OSX/Program.cs

```diff
@@ -14,42 +17,63 @@ class Program : IEnableLogger
         static AppDescOsx _app;
         static ILogger _logger;
 
+        //TODO: IMO it would be better to have there simply be one option for the process to start and a second flag indicating if you should wait
+        //We could then just make the process option required, cleaning up the code and the API a bit.
+        private static readonly Option<string> ProcessStartOption = new("--processStart", "Start an executable in the current version of the app package");
+        private static readonly Option<string> ProcessStartAndWaitOption = new("--processStartAndWait", "Start an executable in the current version of the app package");
+        private static readonly Option<bool> ForceLatestOption = new("--forceLatest", "Force updates the current version folder");
+        private static readonly Option<string> ProcessStartArgsOption = new(new[] { "-a", "--processStartArgs" }, "Arguments that will be used when starting executable");
```

Comment:
IMO it would be better to have there simply be one option for the process to start and a second flag indicating if you should wait
We could then just make the process option required, cleaning up the code and the API a bit.

Reply:
I agree with you in principle, although I believe the choice here was to keep the command line consistent with the equivalent Windows command, so that Squirrel.dll does not need to care if it's executing the windows or macos version. Since UpdateMac is an internal tool at the moment in the OSX version (not meant to be used by application authors) we should be able to update this later without much trouble.

Reply:
Makes sense.